misc/refseq-historical-backfill/uta-extract-historical

```diff
@@ -0,0 +1,52 @@
+#!/usr/bin/env bash
+
+# Download, then extract intermediate files out of the NCBI historical alignment files.
+
+set -e
+
+ncbi_dir=$1
+working_dir=$2
+log_dir=$3
+
+if [ -z "$ncbi_dir" ] || [ -z "$working_dir" ] || [ -z "$log_dir" ]
+then
+    echo 'Usage: sbin/uta-extract-historical <ncbi_dir> <working_dir> <log_dir>'
+    exit 1
+fi
+
+download_ncbi_file () {
```

Comment:
I added download files here so that we did not have to modify the existing NCBI download config.